cli/cmd/extend_baseimage_test.go

```diff
@@ -71,7 +30,7 @@ var _ = Describe("ExtendBaseimageCmd", func() {
 		mockEnv = env.NewMockEnv(GinkgoT())
 		globalOpts = cmd.GlobalOptions{}
 		opts = &cmd.ExtendBaseimageOpts{
-			GlobalOptions: &globalOpts,
+			GlobalOptions: globalOpts,
```

Comment:
Eventually, I think these all need to be pointers so persistentflags are passed through correctly.